src/components/Footer.astro

```diff
@@ -1,270 +1,213 @@
+			<path d="M100.27,97a14.58,14.58,0,0,0,11.15-5.1l2.17,1.92c-3.31,3.57-7.2,6.12-13.45,6.12-8.54,0-15.94-6.83-15.94-17V82.8c0-9.44,6.63-17,15.43-17,9.12,0,14.92,7.4,14.92,17V83a6.68,6.68,0,0,1-.07,1.15h-27C88.09,92.23,93.89,97,100.27,97Zm10.9-15.62c-.45-6.56-4.21-12.62-11.67-12.62-6.44,0-11.41,5.42-12,12.62Z"/>
+			<path d="M132.54,99.76A22.26,22.26,0,0,1,119.21,95l1.85-2.49a19,19,0,0,0,11.67,4.27c4.4,0,7.84-2.42,7.84-6.18v-.13c0-3.82-4.08-5.22-8.61-6.5-5.29-1.53-11.15-3.12-11.15-8.92V75c0-5.23,4.4-9,10.71-9a21.92,21.92,0,0,1,11.54,3.57l-1.66,2.61a18.75,18.75,0,0,0-10-3.25c-4.46,0-7.39,2.42-7.39,5.67v.13c0,3.64,4.39,5,9.05,6.31,5.23,1.47,10.71,3.38,10.71,9.12v.13C143.76,96.06,138.79,99.76,132.54,99.76Z"/>
+			<path d="M164.41,97a14.58,14.58,0,0,0,11.15-5.1l2.17,1.92c-3.31,3.57-7.2,6.12-13.45,6.12-8.54,0-15.94-6.83-15.94-17V82.8c0-9.44,6.63-17,15.43-17,9.12,0,14.92,7.4,14.92,17V83a6.68,6.68,0,0,1-.07,1.15h-27C152.23,92.23,158,97,164.41,97Zm10.9-15.62c-.45-6.56-4.21-12.62-11.67-12.62-6.44,0-11.41,5.42-12,12.62Z"/>
+			<path d="M209.54,99.42V93.7a14.8,14.8,0,0,1-12.43,6.19c-6.06,0-12.56-3.45-12.56-10.21v-.12c0-6.76,5.8-10.59,14.21-10.59a39.05,39.05,0,0,1,10.78,1.41V79c0-6.56-4-9.94-10.71-9.94a21.77,21.77,0,0,0-10.2,2.61l-1.15-2.74A26.1,26.1,0,0,1,199,66.1c8.35,0,13.58,4.33,13.58,12.75V99.42Zm.06-16.17a42.65,42.65,0,0,0-11-1.47c-6.88,0-10.71,3.06-10.71,7.59v.12c0,4.72,4.47,7.46,9.44,7.46,6.63,0,12.3-4.08,12.3-10Z"/>
+			<path d="M239.46,69.54c-7.52,0-14.09,5.67-14.09,16.26V99.42h-3.13V65.94h3.13v9.91c2.55-5.8,8-10,14.41-9.75v3.44Z"/>
+			<path d="M258.6,99.89A16.73,16.73,0,0,1,242,83v-.13c0-9.11,7.27-17,16.57-17,6.12,0,9.95,2.81,13,6l-2.23,2.3c-2.81-2.87-6-5.36-10.84-5.36-7.39,0-13.13,6.19-13.13,14v.12c0,7.84,5.93,14.09,13.38,14.09,4.6,0,8.17-2.36,10.91-5.42l2.17,1.92C268.55,97.21,264.6,99.89,258.6,99.89Z"/>
+			<path d="M303.23,99.42V79.8c0-6.69-3.76-11.09-10.33-11.09-6.37,0-11.28,4.84-11.28,11.6V99.42h-3.13V52.58h3.13V72.66a12.74,12.74,0,0,1,11.73-6.82c8.22,0,13,5.61,13,13.39V99.42Z" />
+		</svg>
+	</section>
+	<section id="land-ack-address">
+	 	<address>
+			Vivic Research<br/>
+			6th floor-123 Slater Street,<br/>
+			Ottawa, Ontario, K1P 5H2<br/>
+			Canada
+		</address>
+		<div id="land-acknowledgement">
+			<p>
+				We acknowledge that our office is on the traditional and unceded
+				Territory of the Anishinabe Algonquin Nation whose land was never
+				given up and never surrendered.
+			</p>
+		</div>
+	</section>
+	<section id="socials-links">
+		<div id="links">
+			<a href="/about">
+				about us
+			</a>
+			<a href="/contact">
+				contact us
+			</a>
+			<a href="/work">
+				our work
+			</a>
+			<a href="/#services">
+				our services
+			</a>
+		</div>
+		<div id="socials">
+			<p>Find us elsewhere:</p>
+			<div id="social-icons">
+				<a
+					aria-label="Instagram"
+					href="https://www.instagram.com/vivicresearch/"
+					class="social-link"
+				>
+					<svg x="0px" y="0px" viewBox="0 0 22 22" role="presentation">
```

Comment:
`role="presentation"` indicated it's not a graphic or diagram. While it sort of _is_ a graphic, the meaning is covered by the aria-label above.